source/core/replica-set-architecture-three-members.txt

```diff
@@ -27,7 +27,7 @@ A replica set with three members that store data has:
 
 - Two :doc:`secondary </core/replica-set-secondary>` members. Both
   secondaries can become the primary in an :doc:`election
-  </core/replica-set-elections>`.
+  </core/replica-set-high-availability>`.
```

Comment:
I know I'm pedantic but I really don't like renaming this url to high availability.  I feel like replica-set-availability would be a much better name; high availability is what you get from using the availability features correctly.